src/.vscode/launch.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file is not necessary.

Reply:
Yeah, this folder was unintentionally commited. Thanks for pointing this out. I'll fix that.